idaes/power_generation/unit_models/boiler_heat_exchanger_2D.py

```diff
@@ -400,6 +399,9 @@ def _make_geometry(self):
                                         mutable=False,
                                         doc='Header wall thickness')
 
+            self.head_ri_scaling = Param(initialize=0.1, mutable=True,
+                                         doc='header inner radius scaling')
```

Comment:
For clarity, is this a numerical scaling factor, or or parameter regarding build up of physical scaling? The doc string does not make this clear.

If it is the first case, we really need to get the scaling tools fully deployed so that we can get rid of numerical scaling parameters.

Reply:
It looks like scaling for the for a differential equation.  Should we try to use the scaling utilities for this, or is there a gap that makes them not work with the DAE stuff?  I'm probably missing something, but just as a matter of style I think we are moving away from scaling parameters like this.